src/db/migrations/20230427152829-init.js

```diff
@@ -2,21 +2,21 @@
 
 const { v4 } = require('uuid');
 
-const userId = v4();
-const adminId = v4();
-const superAdminId = v4();
-
+const userRoleId = v4();
+const adminRoleId = v4();
+const superAdminRoleId = v4();
+const superAdminUserId = v4();
 const roles = [
 	{
-		id: userId,
+		id: userRoleId,
 		type: 'User',
 	},
 	{
-		id: adminId,
+		id: adminRoleId,
 		type: 'Admin',
 	},
 	{
-		id: superAdminId,
+		id: superAdminRoleId,
 		type: 'SuperAdmin',
 	},
 ];
```

Comment:
you don`t need this variables 
const userRoleId = v4();
const adminRoleId = v4();
const superAdminRoleId = v4();
you can do like this
{
		id: v4(),
		id: userRoleId,
		type: 'User',
	}